tensorflow/java/src/main/native/session_jni.cc

```diff
@@ -127,6 +127,8 @@ JNIEXPORT jlong JNICALL Java_org_tensorflow_Session_allocate2(
                  static_cast<size_t>(env->GetArrayLength(config)), status);
     if (!throwExceptionIfNotOK(env, status)) {
       env->ReleaseByteArrayElements(config, cconfig, JNI_ABORT);
+      TF_DeleteSessionOptions(opts);
+      TF_DeleteStatus(status);
```

Comment:
Thanks for the fix!
I think we might also be leaking `ctarget`, perhaps we can move the block that calls `GetStringUTFChars` to fill in `ctarget` after this `if (config != nullptr)` block so that we don't leak it?

Reply:
Thanks for your reminder, this may be the better way. I'll update this commit.